Classes/Controllers/OpenRecentController.m

```diff
@@ -146,8 +145,8 @@ - (NSInteger)numberOfRowsInTableView:(NSTableView *)aTableView
 }
 
 - (IBAction)changeSelection:(id) sender {
-	int i = [resultViewer selectedRow];
-	if(i >= 0 && i < [currentResults count])
+	NSUInteger i = resultViewer.selectedRow;
+	if(i < currentResults.count)
```

Comment:
I think this should be a signed `NSInteger`, as that's what `-[NSTableView selectedRow]` returns. `-1` means no row is selected.